packages/yoroi-extension/app/ergo-connector/stores/ConnectorStore.js

```diff
@@ -297,12 +297,16 @@ export default class ConnectorStore extends Store<StoresMap, ActionsMap> {
       selectedWallet.getParent().getDefaultToken().defaultIdentifier
     ])).filter(id => !mintedTokenIds.includes(id));
     const stateFetcher = this.stores.substores.ergo.stateFetchStore.fetcher;
-    await addErgoAssets({
-      db: selectedWallet.getDb(),
-      tokenIdentifiers,
-      getAssetInfo: stateFetcher.getAssetInfo,
-      network: selectedWallet.getParent().getNetworkInfo(),
-    });
+    try {
+      await addErgoAssets({
+        db: selectedWallet.getDb(),
+        tokenIdentifiers,
+        getAssetInfo: stateFetcher.getAssetInfo,
+        network: selectedWallet.getParent().getNetworkInfo(),
+      });
+    } catch (error) {
+      console.error('Token info unavailable', error.message)
+    }
```

Comment:
Let's not wrap the entire call to the storage with a catch related to unavailable tokens, because there might be a lot of other reasons for a db-call to fail than specifically non-available tokens, so that error message will be very confusing in the logs later.

The state-fetcher function is being passed right there, so you can wrap it into a safety check like that:
```
try {
  await addErgoAssets({
    db: selectedWallet.getDb(),
    tokenIdentifiers,
    getAssetInfo: async (req) => {
        try {
            return await stateFetcher.getAssetInfo(req);
        } catch (e) {
            console.error('Aseet info request failed', e);
        }
    },
    network: selectedWallet.getParent().getNetworkInfo(),
  });
} catch (error) {
  console.error('Failed to add ergo assets!', error);
}
```